source/auth/tests/legacy/connection-string.json

```diff
@@ -89,46 +89,10 @@
         "mechanism": "GSSAPI",
         "mechanism_properties": {
           "SERVICE_NAME": "other",
-          "SERVICE_HOST": "example.com",
-          "CANONICALIZE_HOST_NAME": "forward"
+          "CANONICALIZE_HOST_NAME": true
         }
       }
     },
-    {
-      "description": "should accept forwardAndReverse hostname canonicalization (GSSAPI)",
```

Comment:
why are these tests removed?

Reply:
Same as other comment re DRIVERS-1803

Reply:
Ah, I see the problem.  I copied the file from pymongo where I had added the new tests, but we were not up to date with the spec.  I will revert these changes.

Reply:
Fixed

Reply:
Thanks!